### ISSUE TITLE
test_cgi overrides default trial test timeout - it should not.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11171]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1665|
|Type|defect|
|Created|2006-04-24 01:03:23Z|

UNALLOWED
INAPPROPRIATE

<details><summary>Searchable metadata</summary>

```
trac-id__1665 1665
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web2 web2
keywords__ 
time__1145840603000000 1145840603000000
changetime__1145840697000000 1145840697000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16660]) Remove timeouts from test_cgi

Author: exarkun
Reviewer: itamarst
Fixes [#11171](https://github.com/twisted/twisted/issues/11171)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
